sequins.go

```diff
@@ -150,7 +168,14 @@ func (s *sequins) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	}
 
 	key := strings.TrimPrefix(r.URL.Path, "/")
-	res, err := s.index.Get(key)
+
+	currentIndex := s.indexMonitor.AcquireRef()
+	if currentIndex == nil {
+		log.Fatal("Tried to make HTTP request before initializing the index") 
+	}
```

Comment:
The only case in which this could happen is if there is no initial data (or a programming error). I almost wonder if we shouldn't error out if the initial `refresh()` doesn't find any data in the directory.